backend-go/internal/jwt/middleware.go

```
@@ -0,0 +1,23 @@
+)
+
+type Payload struct {
+	Nickname string `json:"nickname,omitempty"`
+	jwt.RegisteredClaims
+}
+
+// Middleware returns a middleware that checks the JWT token.
+func Middleware(tokenSecret string) echo.MiddlewareFunc {
+	return echojwt.WithConfig(echojwt.Config{
+		SigningKey: []byte(tokenSecret),
+		NewClaimsFunc: func(c echo.Context) jwt.Claims {
```

golangci-lint (1.62.2)

[warning] unused-parameter: parameter 'c' seems to be unused, consider removing or renaming it as _

(revive)